tests/test_smoke.py

```diff
@@ -95,10 +95,7 @@ def __init__(self, name, legality):
     ),
     MockCard( # Illegal in 99 from Unset
         name="Bar Entry", legality=Legality.NOT_LEGAL
-    ),
-    MockCard( # Illegal card type
-        name="Clown Extruder", legality=Legality.NOT_LEGAL
-    ),
```

Comment:
Lgtm but do you think we should just leave this unit test in and change the legality to `Legality.LEGAL`?

Reply:
ooo good call!